### ISSUE TITLE
flag reduction error

### ISSUE THREAD

**Issue by bennahugo** [ratt-ru/CubiCal#367]
```
# INFO      09:15:48 - data_handler       [io] [0.1/0.1 2.3/2.3 21.2Gb] reading FLAG
# INFO      09:15:48 - main               [io] [0.1/0.1 2.3/2.3 21.2Gb] I/O handler for load 0 save None failed with exception: Required argument 'array' (pos 1) not found
# INFO      09:15:48 - main               [io] [0.1/0.1 2.3/2.3 21.2Gb] Traceback (most recent call last):
#   File "/usr/local/lib/python3.6/dist-packages/cubical/workers.py", line 457, in _io_handler
#     tile.load(load_model=load_model)
#   File "/usr/local/lib/python3.6/dist-packages/cubical/data_handler/ms_tile.py", line 774, in load
#     flagcol[:] = np.logical_or.reduce(axis=-2)[:,:,np.newaxis]
```
This is clearly a bug...

**Comment by JSKenyon** [comment]
Absolutely - I had also run into it. It is fixed in #366. That will go in soon.

**Comment by o-smirnov** [comment]
Fixed in #366 already. 

How is it that something as outrageously broken as this sits there unnoticed for months, and then gets spotted by three people on the same morning?
